include/swift/Basic/LangOptions.h

```diff
@@ -274,7 +274,8 @@ namespace swift {
     bool EnableInferPublicSendable = false;
 
     /// Disable the implicit import of the _Concurrency module.
-    bool DisableImplicitConcurrencyModuleImport = false;
+    bool DisableImplicitConcurrencyModuleImport =
+        !SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY;
```

Comment:
Would it be easier to check `SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY` in `shouldImportConcurrencyByDefault` where we make the decision?

Reply:
I don't think it would be easier, and adjusting it in LangOpts makes the default match the build setup. Otherwise, I don't have a strong opinion about this. If people want to debate this in a subsequent pull request, that seems fine. I just want to unbreak minimum dependency builds.